lib/init.js

```diff
@@ -3,7 +3,7 @@ if (module.parent && module.parent.id === 'internal/preload') {
   globalThis.__debug__ = true;
 }
 
-import('./index.mjs')
+import('../dist/index.js')
```

Comment:
Do we need Closure Compiler to be able to handle this?  Does it?

Reply:
We don't - this file is only used to launch the polyfill Playground.